src/components/app/details/triggerView/MaterialSource.tsx

```diff
@@ -69,18 +69,19 @@ export class MaterialSource extends Component<MaterialSourceProps> {
         return (
             <>
                 {this.props.material.map((material, index) => {
-                    let selectedClass = material.isSelected ? 'material-selected' : ''
                     return (
                         <div
                             key={index}
-                            className={`material-list__item ${selectedClass}`}
+                            className={`material-list__item ${material.isSelected ? 'material-selected' : ''}`}
                             onClick={(e) => {
```

Comment:
Please extract this out

Reply:
Done. Also, converted this Class component to FC.